src/spikeinterface/core/recording_tools.py

```diff
@@ -302,7 +302,7 @@ def get_chunk_with_margin(
     return traces_chunk, left_margin, right_margin
 
 
-def order_channels_by_depth(recording, channel_ids=None, dimensions=("x", "y")):
+def order_channels_by_depth(recording, channel_ids=None, dimensions=("x", "y"), flip=False):
     """
     Order channels by depth, by first ordering the x-axis, and then the y-axis.
```

Comment:
Question out of curiosity, do we have a convention that x goes along the width or the recorder and that y goes in the height / depth (the direction where it is usualy moves) or is it the other way around?

I found myself realizing that I did not know what to expect after reading this.

Reply:
Hi @h-mayorquin 

Yes the convention is that `x` is width and `y` is depth. The `('x', 'y')` tuple is fed into `lexsort`, which sorts outer to innermost dimensions ;)

Reply:
Thanks, @alejoe91.